python/tests/test_genotypes.py

```diff
@@ -22,18 +22,16 @@
 """
 Test cases for generating genotypes/haplotypes.
 """
-import unittest
 import itertools
 import random
+import unittest
 
-import numpy as np
 import msprime
-
+import numpy as np
+import tests.test_wright_fisher as wf
```

Comment:
The ``tests`` module probably needs to be registered as local.

Reply:
Looks fixed with `reorder_python_imports`